Dockerfile

```diff
@@ -28,20 +33,21 @@ EXPOSE 5000
 WORKDIR /code
 
 # NOTE: requirements.txt not likely to change between dev builds
-COPY requirements.txt /code/requirements.txt
+COPY requirements.txt .
```

Comment:
See question below. Is this necessary if you're just copying it to the same place?

Reply:
Yeah, basically does the same thing

Reply:
Yeah this Docker syntax is actually <src> <dest>. The <src> is on the localhost and <dst> is inside the container image. This syntax copies the current working directory into the working directory set in the container. Confusing to look at but it isn't the same place. ;)

Reply:
Okay, that makes sense to me, sorry for my ignorance. You learn something every day!

Reply:
Should have read the documentation before commenting. I think I understood this at one point, since I've written a `Dockerfile`, but I guess I forgot.